python/cuml/metrics/cluster/silhouette_score.pyx

```diff
@@ -105,6 +106,16 @@ def _silhouette_coeff(
         labels.to_output(output_type='cupy', output_dtype='int')
     ).shape[0]
 
+    if not check_labels(labels, cp.arange(n_labels, dtype=np.int32)):
```

Comment:
I think adding a test case that exercises this would be a good idea unless there is one already? Didn't see it on a quick look

Reply:
Any suggestions on how to test this without hard-coding some examples?

Reply:
Hard-coding is great! Allows you to exactly control the case. Much nicer than just using some big dataset that's hard to debug.

Reply:
Thanks! I'll just add the examples in the issue